sdk/formrecognizer/azure-ai-formrecognizer/samples/sample_recognize_custom_forms.py

```diff
@@ -0,0 +1,62 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: sample_recognize_custom_forms.py
+
+DESCRIPTION:
+    This sample demonstrates how to analyze a form from a document with a custom
+    trained model. To learn how to train your own models, look at
+    sample_train_unlabeled_model.py and sample_train_labeled_model.py
+USAGE:
+    python sample_recognize_custom_forms.py
+
+    Set the environment variables with your own values before running the sample:
+    1) AZURE_FORM_RECOGNIZER_ENDPOINT - the endpoint to your Cognitive Services resource.
+    2) AZURE_FORM_RECOGNIZER_KEY - your Form Recognizer subscription key
+    3) CUSTOM_TRAINED_MODEL_ID - the ID of your custom trained model
+"""
+
+import os
+
+
+class RecognizeCustomForms(object):
+
+    endpoint = os.environ["AZURE_FORM_RECOGNIZER_ENDPOINT"]
+    key = os.environ["AZURE_FORM_RECOGNIZER_KEY"]
+    model_id = os.environ["CUSTOM_TRAINED_MODEL_ID"]
+
+    def recognize_custom_forms(self):
+        # TODO: this can be used as examples in sphinx
+        from azure.core.credentials import AzureKeyCredential
+        from azure.ai.formrecognizer import FormRecognizerClient
+        form_recognizer_client = FormRecognizerClient(
+            endpoint=self.endpoint, credential=AzureKeyCredential(self.key)
+        )
+        with open("sample_forms/forms/Form_1.jpg", "rb") as f:
```

Comment:
Should we provide a sample file to analyze?
What if the model Id wasn't trained for that form type? It would result in not showing relevant fields?

Reply:
I see what you mean. I'll add a comment stressing that the form type needs to be the same as the ones the custom models are trained on